esphome/components/emporia_vue/emporia_vue.h

```diff
@@ -39,39 +39,48 @@ class CTSensor;
 
 class EmporiaVueComponent : public Component, public i2c::I2CDevice {
  public:
-  void setup() override;
-  void loop() override;
   void dump_config() override;
-  static void i2c_request_task(void *pv);
 
+  void set_sensor_poll_interval(uint32_t sensor_poll_interval) { this->sensor_poll_interval_ = sensor_poll_interval; }
```

Comment:
Moved this default from C++ to Python, suggested [here](https://esphome.io/guides/contributing.html#config-validation)

> All default values should be defined in the schema (and not in C++ codebase or other code parts).